packages/numbers/src/numbers.ts

```diff
@@ -41,3 +41,30 @@ export function clamp(min: number, max: number, value?: number) {
   const doit = (value: number) => Math.min(Math.max(value, min), max);
   return typeof value === 'undefined' ? doit : doit(value);
 }
+
+export const gt = (min: number) => (value: number) => value > min;
+export const lt = (max: number) => (value: number) => value < max;
+export const gte = (min: number) => (value: number) => value >= min;
+export const lte = (max: number) => (value: number) => value <= max;
+export const eq = (value: number) => (other: number) => value === other;
```

Comment:
There is some overlap here with predicates in collections. How would you like me to resolve that?
https://github.com/execonline-inc/CooperTS/blob/main/packages/collections/src/predicates.ts

Reply:
I think it is fine to have both. These are specifically numerical while the ones in collections are more generic.